### ISSUE TITLE
[Suggestion] More AobScan overloads

### ISSUE THREAD

**Issue by marcusx2** [erfg12/memory.dll#68]
`public Task<IEnumerable<long>> AoBScan(long start, string search, bool writable = false, bool executable = true, string file = "");

public Task<IEnumerable<long>> AoBScan(string start, string search, bool writable = false, bool executable = true, string file = "");

public Task<IEnumerable<long>> AoBScan(string start, string end, string search, bool writable = false, bool executable = true, string file = "");
`
Basically, being able to provide just the start address, and also being able to provide the start address as a string, passing in a module name. When doing so, memory.dll will get the start address of the module that I passed in.
